Neos.Neos/Classes/Domain/Import/LiveWorkspaceCreationProcessor.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Import;
+
+use Neos\ContentRepository\Core\ContentRepository;
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\Export\ProcessingContext;
+use Neos\ContentRepository\Export\ProcessorInterface;
+use Neos\ContentRepository\Export\Severity;
+use Neos\Neos\Domain\Model\WorkspaceDescription;
+use Neos\Neos\Domain\Model\WorkspaceRole;
+use Neos\Neos\Domain\Model\WorkspaceRoleAssignment;
+use Neos\Neos\Domain\Model\WorkspaceTitle;
+use Neos\Neos\Domain\Service\WorkspaceService;
+
+/**
+ * Import processor that creates the "live" workspace if it doesn't exist
+ */
+final readonly class LiveWorkspaceCreationProcessor implements ProcessorInterface
+{
+    public function __construct(
+        private ContentRepository $contentRepository,
+        private WorkspaceService $workspaceService,
+    ) {
+    }
+
+    public function run(ProcessingContext $context): void
+    {
+        $context->dispatch(Severity::NOTICE, 'Creating live workspace');
+        $liveWorkspace = $this->contentRepository->findWorkspaceByName(WorkspaceName::forLive());
+        $liveContentStreamVersion = $liveWorkspace ? $this->contentRepository->findContentStreamById($liveWorkspace->currentContentStreamId)?->version : null;
+        if ($liveWorkspace && $liveContentStreamVersion !== 0) {
+            // todo we cannot use `hasPublishableChanges` here... maybe introduce `hasChanges`?
+            throw new \RuntimeException('Live workspace already contains content please run "site:pruneAll" before importing.');
+        }
```

Comment:
todo this is not totally ideal yet. Should we maybe adjust https://github.com/neos/neos-development-collection/pull/5332 to be just `hasChanges`?

Reply:
I think this is no use-case we need to support? IF the workspace already exists we need to abort?